Contentful.Core/Images/ImageUrlBuilder.cs

```diff
@@ -47,6 +47,16 @@ public ImageUrlBuilder SetJpgQuality(int quality)
             return this;
         }
 
+        /// <summary>
+        /// Sets the quality of the png image returned to 8 bit.
```

Comment:
Hmm, that reads counter intuitive. It is not the quality reduced to 8 bit, but rather the colour depth.

Maybe `sets the color depth to 8bit, aka 256 colors`, or something?